src/Summary/Summary.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "styled-components";
+import { pixelDigitsFrom } from "../NavBar/NavBar";
+import { Box } from "../Box";
+import theme from "../theme";
+import { SummaryContextProvider } from "./SummaryContext";
+
+type SummaryProps = {
+  collapseBreakpoint?: number | string;
+};
+
+const SummaryWrapper = styled(Box)<{ collapseBreakpoint: number }>(({ theme, collapseBreakpoint }) => ({
+  display: "flex",
+  background: theme.colors.white,
+  gap: theme.space.x2,
+  paddingTop: theme.space.x1,
+  paddingBottom: theme.space.x1,
+  paddingLeft: theme.space.x2,
+  paddingRight: theme.space.x2,
+  borderRadius: theme.radii.medium,
+  width: "fit-content",
+
+  [`@media (max-width: ${collapseBreakpoint}px)`]: {
+    background: "none",
+    flexFlow: "wrap",
+  },
+}));
+
+const DEFAULT_COLLAPSE_BREAKPOINT = theme.breakpoints.small;
+
+const Summary: React.FC<SummaryProps> = ({ collapseBreakpoint, children }) => {
+  const collapseBreakpointPx = pixelDigitsFrom(collapseBreakpoint);
+
+  return (
+    <SummaryContextProvider collapseBreakpoint={collapseBreakpointPx}>
+      <SummaryWrapper collapseBreakpoint={collapseBreakpointPx}>{children}</SummaryWrapper>
+    </SummaryContextProvider>
+  );
+};
+
+Summary.displayName = "Summary";
+
+Summary.defaultProps = {
+  collapseBreakpoint: DEFAULT_COLLAPSE_BREAKPOINT,
```

Comment:
Does it make sense to move `defaultProps` directly to component definition? Example:
```
const Summary: React.FC<SummaryProps> = ({ collapseBreakpoint = DEFAULT_COLLAPSE_BREAKPOINT, children }) => {
```